samples/apps/react-vanilla-sample/README.md

```
@@ -1,98 +1,66 @@
 # React Vanilla Sample Application
 
-This sample React application demonstrates integrating authentication into your application. It supports two authentication approaches:
+This sample React application demonstrates integrating authentication and registration into your application using the app-native flow orchestration API.
 
-- **Native Flow**: App-native authentication using  flow orchestration API
-- **OAuth 2.0 / OIDC**: Standards-based authentication using OAuth 2.0 Authorization Code flow
+### Supported Authentication Methods
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)